tests/test_spark_distributions.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import os
+import pytest
+import spark_utils as utils
+
+
+log = logging.getLogger(__name__)
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+SMOKE_TEST_SPARK_DOCKER_IMAGES = os.getenv('SMOKE_TEST_SPARK_DOCKER_IMAGES', '')
+
+
+@pytest.mark.sanity
+@pytest.mark.smoke
+def test_spark_docker_images():
+    if not SMOKE_TEST_SPARK_DOCKER_IMAGES:
```

Comment:
👍 in the future instead of publishing jars and including them in `resource.json` we'll publish different docker images so we can utilize this test for all of them